### ISSUE TITLE
Waypointing multiple units results in tons of waypoints

### ISSUE THREAD

**Issue by Madrageeeeee** [xezon/GeneralsGamePatch#668]
I never understood why when you have multiple units selected and you click somewhere to move them there (move command) you see one waypoint highlight. But when you waypoint multiple units you see a waypoint for every unit in that group or at least multiple. I feel it should be just one there as well.

hope this makes sense

**Comment by xezon** [comment]
Makes sense to perhaps just show one waypoint in a group selection instead of all.

**Comment by MTKing4** [comment]
It's correct the way it is and it shouldn't be changed, it's more useful to know where every unit is going to, this is a logical setup and is the norm in every RTS game I've ever played, nothing strange or out of the ordinary about that.
